client/api_bookies.go

```diff
@@ -0,0 +1,336 @@
+/*
+ * Pulsar Admin REST API
+ *
+ * This provides the REST API for admin operations
+ *
+ * API version: v2
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package swagger
```

Comment:
The package name is `swagger` but the folder is called `client`, can we rename the package to `client` or `adminclient`?

Reply:
sure

Reply:
done